server/questions.js

```diff
@@ -96,7 +101,48 @@ exports.updateQuestionById = function(questionId, info, callback) {
 }
 
 var updateQuestionById = function(questionId, info, callback) {
-	db.updateQuestionById(questionId, info, callback);
+	updateQuestionByType(questionId,info,callback);
```

Comment:
updateQuestionByType does not make sense, there are multiple questions of the same type. Lets put the content of updateQuestionByType in this function. That way when you do update question by id , you will do the validation then pass to db.udpateQuestionById

Reply:
Yeah we can combine it into a single function, so no confusion